pkg/pillar/types/domainmgrtypes.go

```diff
@@ -25,6 +26,9 @@ type DomainConfig struct {
 	VifList           []VifInfo
 	IoAdapterList     []IoAdapter
 	CloudInitUserData string // base64-encoded
+	// Container related info
+	IsContainer      bool   // Is this Domain for a Container?
+	ContainerImageID string // SHA-512 of rkt container image
```

Comment:
AFAICT this should be a property in StorageStatus/DiskStatus, since it it per downloaded image and not per container.

Reply:
Currently, 1 Domain = 1 App Instance = 1 Container ( For containers ) = 1 Container Image

So, it is very convenient to have it here. I suggest we leave it here till this relationship changes. Keeps the code simple.